### PR TITLE
Initial version of project description for TAD Project

### PR DESCRIPTION
**Pull request by laurensWe**
# Description

Project Description added for the TAD project with a little FAQ section

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilises.

## Developer Certificate of Origin

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: docs/Projects/TAD/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# TAD
+
+TAD is the abbreviation of Transparency of Algorithmic Decision Making, the goal is to make algorithmic systems more
```
Comment (anneschuth) [inline docs/Projects/TAD/index.md:3]:
```suggestion
TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making, the goal is to make algorithmic systems more
```

File: docs/Projects/TAD/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+transparent by generating standardized reports on the algorithmic system for both it's technical element as well as the
+surrounding system. For both the model and the system the lifecycle is important and this needs to be taken into
+account.
+
+The TAD project is focused on one hand building the standardized format of reporting on an algorithmic system by
+extending on the format of [Industry Standard](Reporting-Standard%2Findex.md). On the other hand the project is focused
```
Comment (anneschuth) [inline docs/Projects/TAD/index.md:9]:
```suggestion
extending on the format of [Industry Standard](Reporting-Standard). On the other hand the project is focused
```

**Review by anneschuth — APPROVED** [review]
Nice, I think this is a good start. Two small comments

File: docs/Projects/TAD/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# TAD
+
+TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making, the goal is to make algorithmic systems more
+transparent by generating standardized reports on the algorithmic system for both it's technical element as well as the
+surrounding system. For both the model and the system the lifecycle is important and this needs to be taken into
+account.
```
Comment (ChristopherSpelt) [inline docs/Projects/TAD/index.md:6]:
```suggestion
TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making. TAD has the goal to make algorithmic systems more transparent; it achieves this by generating standardized reports on the algorithmic system which encompasses both technical aspects as well as descriptive information about the system and regulatory assessments. For both the system and the model the lifecycle is important and this needs to be taken into
account.
```

File: docs/Projects/TAD/index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# TAD
+
+TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making, the goal is to make algorithmic systems more
+transparent by generating standardized reports on the algorithmic system for both it's technical element as well as the
```
Comment (ChristopherSpelt) [inline docs/Projects/TAD/index.md:4]: Also we should maybe add some explenations about the used terminology. For example: "An algorithmic system is a system consisting out of one or more ML models".

File: docs/Projects/TAD/index.md (hunk 3 above)
Comment (ChristopherSpelt) [inline docs/Projects/TAD/index.md:6]: The last sentence of this paragraph does not really naturally follow from the previous sentences. Also it feels unfinished.

File: docs/Projects/TAD/index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+surrounding system. For both the model and the system the lifecycle is important and this needs to be taken into
+account.
+
+The TAD project is focused on one hand building the standardized format of reporting on an algorithmic system by
+extending on the format of [Industry Standard](Reporting-Standard). On the other hand the project is focused
+on filling in the system- and model cards with both technical metrics & measurements of the exact model as theoretical
+assessments on the system where the specific algorithm resides. The requirements and instruments are dictated by the
+[Algoritmekader](https://minbzk.github.io/Algoritmekader/).
```
Comment (ChristopherSpelt) [inline docs/Projects/TAD/index.md:12]:
```suggestion
On the one hand the TAD project is focused on building a standardized format of reporting on a algorithmic system by developing a [Reporting Standard](Reporting-Standard). This Reporting Standard consists out of a System Card which contains Model Cards and Assessment Cards. On the other hand the project is focused
on filling in the System, Model and Assessment Cards with both performance metrics and technical measurements on fairness and bias of the model and 
assessments on the system where the specific algorithm resides. The requirements and instruments are dictated by the
[Algoritmekader](https://minbzk.github.io/Algoritmekader/).
```

File: docs/Projects/TAD/index.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+## FAQ
+
+Is TAD an AI Governance Platform?
+
+While TAD will generate standardized reporting, and you are able to edit reporting of algorithmic systems the
```
Comment (ChristopherSpelt) [inline docs/Projects/TAD/index.md:18]:
```suggestion
While TAD will generate a standardized report and you are able to edit report of algorithmic systems, the
```

File: docs/Projects/TAD/index.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+governance/management aspect of different models is currently out of scope for this project. To keep track, compare,
+and alert on models a different approach should be taken but could very much be the follow-up from this project.
+
+Is TAD a tool?
+
+While the project does not aim to be just another tool, the current tools we have compared in the market are
+shortcoming therefore with ADR (XXX) we are proposing to make a set of tools which will help us realize the goal
+of filling in the standardized reporting format for algorithmic systems.
```
Comment (ChristopherSpelt) [inline docs/Projects/TAD/index.md:26]:
```suggestion
While the project does not aim to be just another tool, the existing tools we have compared in the market are
not sufficient to our needs. Therefore, with ADR (XXX) we are proposing to make a set of tools which will help us realize the goal
of filling in the standardized reporting format for algorithmic systems.
```

**Review by ChristopherSpelt — COMMENTED** [review]
Nice start! I added some suggestions.

File: docs/Projects/TAD/index.md (hunk 4 above)
Comment (laurensWe) [reply docs/Projects/TAD/index.md]: I just added a reference to the definition of an algorithm of the algoritme register, would this suffice or would you like to descripe also the algorithm system better?

File: aivalidation_dictionary.txt [hunk 8]
```diff
@@ -94,3 +95,6 @@ webex
 webserver
 Weijs
 Woudloper
+ransparency
```
Comment (laurensWe) [inline aivalidation_dictionary.txt:98]: @berrydenhartog what do you think about this? Or should I remove the capitalized bolded text in the project description itself?

**Review by robbertbos — CHANGES_REQUESTED** [review]

File: docs/Projects/TAD/index.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making. TAD has the goal to make algorithmic
+systems more transparent; it achieves this by generating standardized reports on the algorithmic system which
+encompasses both technical aspects in addition to descriptive information about the system and regulatory assessments.
+For both the system and the model the lifecycle is important and this needs to be taken into account. The definition
+for an algorithm is derived from the [Algoritmeregister](https://algoritmes.overheid.nl/nl/footer/over-algoritmes).
+
+On the one hand the TAD project is focused on building a standardized format of reporting on a algorithmic system by
+developing a [Reporting Standard](Reporting-Standard). This Reporting Standard consists out of a System Card which
+contains Model Cards and Assessment Cards. On the other hand the project is focused on filling in the System, Model and
```
[the remark this answers is not in the document]
Comment (laurensWe) [reply docs/Projects/TAD/index.md:11]: ah thank you, yeah I thought I knew this though. As this is also a standard markdown feature. But extra thanks for your review!

**Review by robbertbos — APPROVED** [review]

**Pull request merged**